test/BigInt/global_object.js

```diff
@@ -0,0 +1,21 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+
+if (this.WScript && this.WScript.LoadScriptFile) { // Check for running in ch
+    this.WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+}
+
+var tests = [
+    {
+        name: "Test BigInt global object properties",
+        body: function () {
+            assert.isTrue(BigInt.length == 1);
+            assert.isTrue(BigInt.name == "BigInt");

```

Comment:
while we are at it, check the prototype too?

Reply:
Can we get the property descriptor here and check writable === false, enumerable === false, configurable === false?